### PR TITLE
Added navbar component

### PR DESCRIPTION
**Pull request by Mayureshkaulwar**
i have added navbar component and please accept my pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,6 +72,21 @@
       href="assets/img/CALCULATOR.png"
       alt="CALCULATOR"
     />
+    <nav class="navbar">
+      <div class="brand-title">CALCULATOR | YOSHITHA</div>
```
Comment (YoshithaRathnayake) [inline index.html:76]: Don't change the font style of the main heading @Mayureshkaulwar

Comment (Mayureshkaulwar) [reply index.html:76]: Ok

Comment (Mayureshkaulwar) [reply index.html:76]: Can I create new pull request?

Comment (YoshithaRathnayake) [reply index.html:76]: Sure @Mayureshkaulwar

Comment (Mayureshkaulwar) [reply index.html:76]: I have created new pull request, please merge that

**Pull request closed without merging**